marketplaceItems/drawApp/fingerDraw_app.js

```diff
@@ -0,0 +1,596 @@
+                        'strokeWidths', 'age']);
+                    var linePointsCount = lineProperties.linePoints.length;
+                    if (linePointsCount > MAX_LINE_POINTS) {
+                        var lastPointDisplacement = lineProperties.linePoints[linePointsCount - 1];
+                        linePoints = [lastPointDisplacement, displacementFromStart];
+                        lineNormals = [DEFAULT_NORMAL, DEFAULT_NORMAL];
+                        if (paintSphere) {
+                            paintSphereDimensions = Entities.getEntityProperties(paintSphere, 'dimensions').dimensions;
+                        }
+                        lineStrokeWidths = [paintSphereDimensions.x, paintSphereDimensions.x];
+                        polyLine = Entities.addEntity({
+                            type: "PolyLine",
+                            name: "Draw App Polyline",
+                            position: previousLinePoint,
+                            linePoints: linePoints,
+                            normals: lineNormals,
+                            strokeWidths: lineStrokeWidths,
+                            color: { red: 255, green: 255, blue: 255 },
+                            isUVModeStretch: true,
+                            lifetime: DECAY_TIME_S,
+                            collisionless: true,
+                            faceCamera: true
+                        }, 'avatar');
+                    } else {
+                        if (paintSphere) {
+                            paintSphereDimensions = Entities.getEntityProperties(paintSphere, 'dimensions').dimensions;
+                        }
+                        lineProperties.linePoints.push(displacementFromStart);
+                        lineProperties.normals.push(DEFAULT_NORMAL);
+                        lineProperties.strokeWidths.push(paintSphereDimensions.x);
+                        Entities.editEntity(polyLine, {
+                            linePoints: lineProperties.linePoints,
+                            normals: lineProperties.normals,
+                            strokeWidths: lineProperties.strokeWidths,
+                            lifetime: lineProperties.age + DECAY_TIME_S
+                        });
+                    }
+                }
+            }
+        }, REPEAT_DISTANCE_CHECK_MS);
+    }
+   
+
+    /* ON TRIGGER RELEASE DRAW: Stop checking distance hand has moved and update current polyline lifetime to decay in 1 
+    minute. */
+    function triggerReleased() {
+        if (activeTriggerPress) {
+            activeTriggerPress = false;
+            if (distanceCheckInterval) {
+                Script.clearInterval(distanceCheckInterval);
+                distanceCheckInterval = null;
+            }
+            if (injector) {
+                injector.stop();
+                injector = null;
+            }
+            polyLine = null;
```

Comment:
these four lines are the same as on line 389 - could you please make a function (perhaps `resetPolyLine()` or something?) that does these four things and call that here and on line 389?